archive-index.html

```diff
@@ -0,0 +1,897 @@
+                  You may stand upon me, <br>
+                  But do not hide your face.”
+                </p>
+                
+<!--  Modal -->              
+                <button class="btn btn-primary" data-toggle="modal" data-target="#poem2">Read More <span class="glyphicon glyphicon-chevron-right"></span></button>
+
+                <div class="modal fade" id="poem2" tabindex="-1" role="dialog" aria-labelledby="myModalLabel" aria-hidden="true">
+                  <div class="modal-dialog" role="document">
+                    <div class="modal-content">
+                      <div class="modal-header">
+                        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                          <span aria-hidden="true">&times;</span>
+                        </button>
+                        <h2 class="modal-title" id="myModalLabel">On the Pulse of Morning</h2>
+                      </div>
+                      <div class="modal-body">
+                        A Rock, A River, A Tree <br>
+                        Hosts to species long since departed,   <br>
+                        Marked the mastodon, <br>
+                        The dinosaur, who left dried tokens   <br>
+                        Of their sojourn here <br>
+                        On our planet floor, <br>
+                        Any broad alarm of their hastening doom   <br>
+                        Is lost in the gloom of dust and ages. <br>
+                        <br>
+                        But today, the Rock cries out to us, clearly, forcefully,   <br>
+                        Come, you may stand upon my <br>
+                        Back and face your distant destiny, <br>
+                        But seek no haven in my shadow, <br>
+                        I will give you no hiding place down here. <br>
+                        <br>
+                        You, created only a little lower than <br>
+                        The angels, have crouched too long in   <br>
+                        The bruising darkness <br>
+                        Have lain too long <br>
+                        Facedown in ignorance, <br>
+                        Your mouths spilling words <br>
+                        Armed for slaughter. <br>
+                        <br>
+                        The Rock cries out to us today,   <br>
+                        You may stand upon me,   <br>
+                        But do not hide your face. <br>
+                      </div>
+                      <div class="modal-footer">
+                        <p class="text-muted">
+                          Maya Angelou, “On the Pulse of Morning” (excerpt) from On the Pulse of Morning. Copyright © 1993 by Maya Angelou.
+                        </p>
+                        <button type="button" class="btn btn-secondary" data-dismiss="modal">Close</button>
+                        <button type="button" class="btn btn-primary">Share</button>
```

Comment:
Again, I appreciate incorporating a Share button as a place holder for work in future sprints to incorporate the social media functionality.